### PR TITLE
Common solution usage

### PR DESCRIPTION
**Pull request by MichielOda**

### CODE REVIEW

File: Parsers.Common/VisualStudio/Projects/IProjectParser.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@ namespace Skyline.DataMiner.CICD.Parsers.Common.VisualStudio.Projects
 {
     internal interface IProjectParser
     {
+        ProjectType GetProjectType();
```
Comment (janstaelensskyline) [inline Parsers.Common/VisualStudio/Projects/IProjectParser.cs:9]: Project Type is associated more with the 'type' of project you can select in the creation wizard of visual studio when adding new projects.
I suggest to use ProjectStyle GetProjectStyle(); to better differentiate between the 'style: SDK/Legacy' and 'type' of the projects.

File: Parsers.Common/VisualStudio/Projects/ProjectType.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿namespace Skyline.DataMiner.CICD.Parsers.Common.VisualStudio.Projects
+{
+    /// <summary>
+    /// Type of the project.
+    /// </summary>
+    public enum ProjectType
```
Comment (janstaelensskyline) [inline Parsers.Common/VisualStudio/Projects/ProjectType.cs:6]: As indicated before, I would suggest ProjectStyle  here.

**Review by janstaelensskyline — COMMENTED** [review]
Please check the suggestion on the naming.

File: Parsers.Common/VisualStudio/Projects/IProjectParser.cs (hunk 1 above)
Comment (MichielOda) [reply Parsers.Common/VisualStudio/Projects/IProjectParser.cs:9]: Done

File: Parsers.Common/VisualStudio/Projects/ProjectType.cs (hunk 2 above)
Comment (MichielOda) [reply Parsers.Common/VisualStudio/Projects/ProjectType.cs:6]: Done

**Review by janstaelensskyline — APPROVED** [review]

**Pull request merged**